examples/enricher/src/test/java/org/apache/apex/examples/enricher/ApplicationTest.java

```diff
@@ -46,6 +56,36 @@ public void testApplication() throws Exception
     } catch (ConstraintViolationException e) {
```

Comment:
The check is great, can you also remove the timing dependency? Instead of running for 10s, have it run until the expected result appears (check in loop) or timeout. See https://github.com/apache/apex-malhar/blob/master/examples/exactly-once/src/test/java/org/apache/apex/examples/exactlyonce/ExactlyOnceJdbcOutputTest.java#L118

Perhaps also use the new embedded mode API and get rid of the exception catch.